resources/nspawn.go

```diff
@@ -221,10 +238,10 @@ func (obj *NspawnRes) CheckApply(apply bool) (checkOK bool, err error) {
 		}
 	}
 	if obj.State == stopped {
```

Comment:
As per the improved code, can both of these branches be collapsed into a single one?